contracts/vault/AssetManagementDai.sol

```diff
@@ -8,6 +8,8 @@ contract AssetManagementDai is AssetManagement {
     using SafeERC20Upgradeable for IERC20Upgradeable;
 
     uint256 public constant getVersion = 2_000;
+    uint256 private constant _SUPPORTED_NUMBER_OF_STRATEGIES = 3;
```

Comment:
Use this in inside `_getNumberOfSupportedStrategies()`

Reply:
maybe it should be public ?

Reply:
Redundant constant are removed